java-bootcamp-bahia-t1/src/MailListSubject.java

```diff
@@ -0,0 +1,9 @@
+public interface MailListSubject {

```

Comment:
:+1: (but have in mind that if a similar feature is requested in the final project it should be sending an email with the information to the manager/s)